tests/_files/source_for_branched_exec_lines.php

```diff
@@ -460,12 +460,18 @@ public function myMethod()
                 {
                     return;                 // +3
                 }
-            }                               // -3
+
+                public function m1(): void {} // +1
```

Comment:
please add a test for a method in non-anonymous class as well (at least in this test file it is not tested currently)

Reply:
Done :+1: